oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/CoreModuleProvider.java

```diff
@@ -282,14 +282,15 @@ public void start() throws ModuleStartException {
             throw new ModuleStartException(e.getMessage(), e);
         }
 
+        Address gRPCServerInstanceAddress = new Address(moduleConfig.getGRPCHost(), moduleConfig.getGRPCPort(), true);
+        TelemetryRelatedContext.INSTANCE.setId(gRPCServerInstanceAddress.toString());
```

Comment:
As you have moved `#setId` here, please remove all old ones in the coordinator implementations.

Reply:
Removed.